<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PerioTestClass</class>
 <widget class="QWidget" name="PerioTestClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>615</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PerioTest</string>
  </property>
  <widget class="PerioSurfaceHeader" name="toothHeader" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1101</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="PerioSurfaceData" name="perioData" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>1101</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="PerioSurfaceData" name="perioData_2" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>1101</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>1101</width>
     <height>302</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>78</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="PerioView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>495</width>
        <height>300</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>PerioSurfaceData</class>
   <extends>QWidget</extends>
   <header>widgets/PerioSurfaceData.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PerioSurfaceHeader</class>
   <extends>QWidget</extends>
   <header>widgets/PerioSurfaceHeader.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PerioView</class>
   <extends>QGraphicsView</extends>
   <header>PerioView/perioview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="PerioTest.qrc"/>
 </resources>
 <connections/>
</ui>
